views/clucks/index.ejs

```diff
@@ -0,0 +1,55 @@
+<%- include('../partials/header') %>
+
+<div class="clucks-list">
+
+  <% for (let cluck of clucks) {
+
+<!-- Calculate time -->
```

Comment:
Note: As this is all in the for loop, it reads better if the indentation implies that to the reader.

eg.
```
for (let cluck of clucks) {
  <!-- Calculuate time -->
  function timeSince(date) {
    var seconds = Math.floor(...
```